src/components/Icon/Icon.tsx

```diff
@@ -32,39 +70,49 @@ export function Icon({source, color, backdrop, accessibilityLabel}: Props) {
     );
   }
 
+  if (
+    color &&
+    sourceType === 'external' &&
+    newDesignLanguage === true &&
+    NEW_DESIGN_LANGUAGE_COLORS.includes(color)
+  ) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'Recoloring external SVGs is not supported with colors in the new design langauge. Set the intended color on your SVG instead.',
+    );
+  }
```

Comment:
We will not support `filters` for colors in the new design language, so recoloring icons is not supported with these new colors.